timelock-impl/src/main/java/com/palantir/atlasdb/timelock/auth/impl/AuthRequirer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.auth.impl;
+
+import java.util.Map;
+
+import com.palantir.lock.TimelockNamespace;
+import com.palantir.atlasdb.timelock.auth.api.Client;
+import com.palantir.atlasdb.timelock.auth.api.NamespaceMatcher;
+
+interface AuthRequirer {
+    AuthRequirer ALWAYS_REQUIRE = ignored -> true;
```

Comment:
when do you use these? outside of this PR of course

Reply:
that is both `ALWAYS` and `NEVER`

Reply:
this will be used to configure authorizer for timelock 0.x and timelock 1.x; that is 0.x will be privilege based, 1.x will be always_require - and there will be a break-glass flag for both for a while that will set it to never_require. I am changing the structure here; with better documentation.